lisp/plugins/gst_backend/default.json

```diff
@@ -1,5 +1,7 @@
 {
-  "_version_": "2",
+  "_version_": "3",
   "_enabled_": true,
+  "audio-pipeline": ["Volume", "Equalizer10", "DbMeter", "AutoSink"],
+  "video-pipeline": ["Volume", "DbMeter", "Equalizer10", "WaylandSink", "AutoSink"],
   "pipeline": ["Volume", "Equalizer10", "DbMeter", "AutoSink"]
```

Comment:
The `pipeline` key is still read and written by the application preferences dialog ("File" > "Preferences" then "Plugins" > "GstBackend"), where it is intended to be used to allow users to specifiy what the default pipeline for new Media (Audio) Cues should be.

However, as this key is no longer used by the `GstMediaCue` class to build the initial pipeline, this no longer works as intended.